doc/rbd/iscsi-overview.rst

```diff
@@ -4,19 +4,18 @@
 Ceph iSCSI Gateway
 ==================
 
-The iSCSI Gateway presents
-a Highly Available (HA) iSCSI target that exports RADOS Block Device (RBD) images
-as SCSI disks. The iSCSI protocol allows clients (initiators) to send SCSI commands
-to storage devices (targets) over a TCP/IP network, enabling clients without
-native Ceph client support to access Ceph block storage.  These include
-Microsoft Windows and even BIOS.
-
-Each iSCSI gateway exploits the Linux IO target kernel subsystem (LIO) to provide
-iSCSI protocol support. LIO utilizes userspace passthrough (TCMU) to interact
-with Ceph's librbd library and expose RBD images to iSCSI clients. With Ceph’s
-iSCSI gateway you can provision a fully integrated block-storage
-infrastructure with all the features and benefits of a conventional Storage Area
-Network (SAN).
+The iSCSI Gateway presents a Highly Available (HA) iSCSI target that exports
+RADOS Block Device (RBD) images as SCSI disks. The iSCSI protocol allows
+clients (initiators) to send SCSI commands to storage devices (targets) over a
+TCP/IP network, enabling clients without native Ceph client support to access
+Ceph block storage.  
+
+Each iSCSI gateway exploits the Linux IO target kernel subsystem (LIO) to
```

Comment:
s/exploits/uses/

ISTR I wrote "exploits" once and you preferred a different verb ;)